src/components/UI/docs/DocumentNav.tsx

```diff
@@ -32,13 +32,14 @@ export const DocumentNav: FC<Props> = ({ content }) => {
               <Text
                 color={activeHash === heading?.headingId ? 'body' : 'primary'}
                 textStyle='document-nav-link'
+                mb='14px'
```

Comment:
```suggestion
```

I think this can be removed and just update the `document-nav-link` style in `textStyles.ts`

Can be updated to: 
```
'document-nav-link': {
    fontFamily: 'body',
    fontWeight: 400,
    fontSize: '13px',
    lineHeight: 5,
    letterSpacing: '1%',
    mb: 3.5
  },
  ```
  
  3.5 is 14px so using that. Its currently 4 and that should be updated.

Reply:
Not necessarily against, but we don't declare margins in any of the other textStyles, which is why I left this local to the component. Do we want to be moving margins to this file? In this case it doesn't seem to make a difference since the `document-nav-link` style is only used here, but just noting.

As an alternative:
```suggestion
                mb={3.5}
```

Reply:
Ah, okay yeah, overlooked that the `mb: 4` was already there... but yeah, similar to last comment, I think it makes a bit more sense to not mix in margins to the textStyles file unless we're going to be more consistent with it. ... I'll add a commit and push.

Reply:
nm... would approve but it's my PR